webui/settings.py

```diff
@@ -87,7 +87,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': ':memory:',
+        'NAME': os.path.join('/db', 'db.sqlite3'),
```

Comment:
```suggestion
        'NAME': os.path.join(BASE_DIR, 'db', 'db.sqlite3'),
```
And I think it is worth to create a empty directory `db/` in the root of this git repo.